### PR TITLE
Add domains

### PR DESCRIPTION
**Pull request by AntoineMille**

### CODE REVIEW

File: contactform.php [hunk 1]
```diff
@@ -165,17 +165,17 @@ public function sendMessage()
         $message = Tools::getValue('message');
 
         if (!($from = trim(Tools::getValue('from'))) || !Validate::isEmail($from)) {
-            $this->context->controller->errors[] = $this->l('Invalid email address.');
+            $this->context->controller->errors[] = $this->trans('Invalid email address.', array(), 'Modules.Contactform.Shop');
```
Comment (AlexEven) [inline contactform.php]: Shop.Notifications.Error

File: contactform.php [hunk 2]
```diff
@@ -165,17 +165,17 @@ public function sendMessage()
...
         } elseif (!$message) {
-            $this->context->controller->errors[] = $this->l('The message cannot be blank.');
+            $this->context->controller->errors[] = $this->trans('The message cannot be blank.', array(), 'Modules.Contactform.Shop');
```
Comment (AlexEven) [inline contactform.php]: Shop.Notifications.Error

File: contactform.php [hunk 3]
```diff
@@ -165,17 +165,17 @@ public function sendMessage()
...
         } elseif (!Validate::isCleanHtml($message)) {
-            $this->context->controller->errors[] = $this->l('Invalid message');
+            $this->context->controller->errors[] = $this->trans('Invalid message');
```
Comment (AlexEven) [inline contactform.php]: Shop.Notifications.Error

**Comment by maximebiloe** [comment]
Thanks @AntoineMille

**Pull request merged**